### PR TITLE
Add persistent storage for the comments

### PR DESCRIPTION
**Pull request by vdrelciuc**
Comments can now be stored persistently using Google Datastorage. Post requests add new comments to the database, get requests load all the comments.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,18 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
```
Comment (nitinms-sps-program) [inline portfolio/pom.xml:28]: indentation.

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class Comment {
+    private String author;
+    private String content;
+
+    public Comment(String author, String content) {
+        this.author = author;
```
Comment (nitinms-sps-program) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:8]: indentation

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,75 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
```
Comment (nitinms-sps-program) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:22]: ordering

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,56 +2,75 @@
 <html>
 
 <head>
-  <meta charset="UTF-8">
-  <title>My Portfolio</title>
-  <link rel="stylesheet" href="style.css">
-  <script src="script.js"></script>
+	<meta charset="UTF-8">
```
Comment (nitinms-sps-program) [inline portfolio/src/main/webapp/index.html:5]: 2 space indentation

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -2,16 +2,39 @@
  * Adds a random fact about me to the page.
  */
 function addRandomFact() {
-  const facts =
-      ['The only music instrument I ever played was flute and I was terrible at it.', 
-      'I wrote my first \'Hello World\' when I was 12.', 
-      'My Spotify playlist is over 36 hours long.', 
-      'I speak English, French and Romanian. I\'m also learning Spanish!'];
+    const facts =
```
Comment (nitinms-sps-program) [inline portfolio/src/main/webapp/script.js:5]: 2 space indentation

File: portfolio/pom.xml (hunk 1 above)
Comment (vdrelciuc) [reply portfolio/pom.xml]: Fixed in commit a410495

File: portfolio/src/main/java/com/google/sps/data/Comment.java (hunk 2 above)
Comment (vdrelciuc) [reply portfolio/src/main/java/com/google/sps/data/Comment.java]: Fixed in commit a410495

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (vdrelciuc) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Fixed in commit a410495

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (vdrelciuc) [reply portfolio/src/main/webapp/index.html:5]: Fixed in commit a410495

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (vdrelciuc) [reply portfolio/src/main/webapp/script.js]: Fixed in commit a410495

**Comment by vdrelciuc** [comment]
All indentation, import ordering and formatting issues should be fixed with commit a410495

**Pull request merged**